src/components/ModelSteps/ScanResult/ScanResultDialog.jsx

```diff
@@ -110,6 +111,14 @@ const ScanResultDialog = ({
     )
   }
   const SubmitButtonComponent = isLastStep() ? SubmitButton : null
+  const isEditStatusChanged =
+    !isEdit || (isEdit && currentFile.name !== formData.file.name)
+
+  const tooltipKey = t(
+    `Acquisition.${
+      isPDFFile(currentFile) || !isEditStatusChanged ? 'pdf' : 'image'
```

Comment:
Why do we check here `!isEditStatusChanged`

Reply:
Good catch, it's not a necessary condition since all MyPapers files are PDFs. 👍